_data/version2-english/glossary.yml

```diff
@@ -1,4 +1,4 @@
-All Theoretical REP: <i>under construction</i>
+All Theoretical REP: The maximum possible total supply for a certain version of  <a href="#Reputation_Token">REP</a>. That means that the sum of the amount of REP existing in a certain <a href="#Universe">universe</a> and the amount of REP not existing in the same universe but migratable to the universe.<br><br>Depending on the situation, there are three patterns to calculate the value:<ol><li>If no <a href="#Fork" >fork</a> has occurred in the universe where <a href="#REPv2">REPv2</a> exists, then the all theoretical REP in the universe is the sum of the amount of REPv2 and the amount of <a href="#REPv1">REPv1</a> (since REPv1 can migrate to the universe where REPv2 exists at any time). That is 11 million REP.</li><li>If a fork has occurred and the <a href="#Forking_Period">forking period</a> ends, then the all theoretical REP in the <a href="#Child_Universe">child universe</a> is just the amount of REP in the child universe (since REP in the <a href="#Parent_Universe">parent universe</a> cannot be migrated after the forking period).</li><li>If a fork has occurred and the forking period is not over yet, then the all theoretical REP in the child universe is the sum of the amount of REP in the child universe and the all theoretical REP in the parent universe (since REP in the parent universe can be migrated during the forking period).</li></ol>This value is used to calculate the <a href="#Forking_Threshold">forking threshold</a>.
```

Comment:
```suggestion
Theoretic REP Supply: The maximum possible total supply for a certain version of  <a href="#Reputation_Token">REP</a>. This is the sum of REP existing in a particular <a href="#Universe">universe</a> and any REP that is eligible for migration into that universe.
<br>
<br>
Depending on the situation, there are three ways to calculate the value:
<ol>
	<li>If no <a href="#Fork" >fork</a> has occurred in the universe or a fork has started but not completed, then the theoretic REP supply in the universe is the sum of the REP in that Universe and the theoretic REP supply in the previous Augur version (since REP can migrate to from a previous version to the next version at any time prior to a fork). For Augur v2, this totals to 11,000,000 REP minus any that has been burned during v2 disputes.</li>
	<li>If a fork has occurred and the <a href="#Forking_Period">forking period</a> has ended, then the theoretic REP supply in the universe is just the amount of REP in that universe (since neither REP in the <a href="#Parent_Universe">parent universe</a> nor REP in previous versions can be migrated after a fork completes).</li>
</ol>
This value is used to calculate the <a href="#Forking_Threshold">forking threshold</a>.
```

Reply:
Thanks!🌝
To break strings over multiple lines in a YML file, it seems you have to use some notation. (https://yaml-multiline.info/). So this change (=multiple lines) will not work. I'm going to commit this change and after that, I will submit another pull request for multiple lines.

Reply:
Oh, I could not commit your suggestion on this page, since I already merged this pull request. I will reflect your suggestion in another pull request.

Reply:
And it seems tab characters cannot be used in a YML file... I dislike YML.

![image](https://user-images.githubusercontent.com/38453357/107007133-74c9a880-67d5-11eb-8e03-ac41d2b46612.png)

Reply:
I will change:
`there are three ways to calculate the value` 
↓
` there are two ways to calculate the value`

Reply:
I reflected the suggestion by this PR https://github.com/msagansk/Augur.Guide/pull/66

Reply:
Ugh, I didn't realize this was a yaml file.  😢

You can probably keep the newlines and replace the tabs with spaces.  You may need some special indicator though to enter "multiline string mode".

Reply:
> You can probably keep the newlines and replace the tabs with spaces. You may need some special indicator though to enter "multiline string mode".

Yes, the PR https://github.com/msagansk/Augur.Guide/pull/66 includes that 🙂